Packs/NozomiNetworks/TestPlaybooks/playbook-NozomiGuardian_Test.yml

```
@@ -0,0 +1,228 @@
+id: NozomiGuardian-Test
+name: NozomiGuardian-Test
+version: -1
+fromversion: 4.5.0
```

integration has `fromversion: 5.0.0`
```suggestion
fromversion: 5.0.0
```